src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsPremiumPrerollOasis.java

```diff
@@ -21,7 +22,12 @@
   )
   public void adsPremiumPrerollOasis(String wikiName, String article) {
     String testedPage = urlBuilder.getUrlForPath(wikiName, article);
+    testedPage = urlBuilder.appendQueryStringToURL(
+        testedPage,
+        "InstantGlobals.wgAdDriverMegaAdUnitBuilderForFVCountries=[ZZ]"
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "InstantGlobals.wgAdDriverMegaAdUnitBuilderForFVCountries=[ZZ]" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)